From fe0b1d149762eab19bdd3a8018afdf190364ddf3 Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Mon, 20 May 2024 11:09:38 -0700
Subject: [PATCH 3/3] Remove BUILD_EPOCH definitions from RTC inf files

as we will pass that flag to all compilations via an environment variable.

Upstream-Status: Inappropriate [OE specific]

Signed-off-by: Matt Madison <matt@madison.systems>
---
 .../Library/MaximRealTimeClockLib/MaximRealTimeClockLib.inf  | 4 ----
 .../NuvotonRealTimeClockLib/NuvotonRealTimeClockLib.inf      | 5 -----
 2 files changed, 9 deletions(-)

diff --git a/Silicon/NVIDIA/Library/MaximRealTimeClockLib/MaximRealTimeClockLib.inf b/Silicon/NVIDIA/Library/MaximRealTimeClockLib/MaximRealTimeClockLib.inf
index 4a69ad62..64e76bee 100644
--- a/Silicon/NVIDIA/Library/MaximRealTimeClockLib/MaximRealTimeClockLib.inf
+++ b/Silicon/NVIDIA/Library/MaximRealTimeClockLib/MaximRealTimeClockLib.inf
@@ -52,7 +52,3 @@
   gEfiVariableArchProtocolGuid
   AND
   gEfiVariableWriteArchProtocolGuid
-
-# Current usage of this library expects GCC in a UNIX-like shell environment with the date command
-[BuildOptions]
-  GCC:*_*_*_CC_FLAGS = -DBUILD_EPOCH=`date +%s`
diff --git a/Silicon/NVIDIA/Library/NuvotonRealTimeClockLib/NuvotonRealTimeClockLib.inf b/Silicon/NVIDIA/Library/NuvotonRealTimeClockLib/NuvotonRealTimeClockLib.inf
index db01f928..f444c2b8 100644
--- a/Silicon/NVIDIA/Library/NuvotonRealTimeClockLib/NuvotonRealTimeClockLib.inf
+++ b/Silicon/NVIDIA/Library/NuvotonRealTimeClockLib/NuvotonRealTimeClockLib.inf
@@ -52,8 +52,3 @@
   gEfiVariableArchProtocolGuid
   AND
   gEfiVariableWriteArchProtocolGuid
-
-# Current usage of this library expects GCC in a UNIX-like shell environment with the date command
-# BUILD_EPOCH will be used as the reference date/time needed for Virtual RTC feature.
-[BuildOptions]
-  GCC:*_*_*_CC_FLAGS = -DBUILD_EPOCH=`date +%s`
-- 
2.34.1

